R/init.R

```diff
@@ -136,6 +139,10 @@ init <- function(data,
     modules <- do.call(teal::modules, modules)
   }
 
+  landing <- extract_landing(modules)
+  if (length(landing) > 1L) stop("teal only supports apps with one module of `tm_landing_poup` class.")
+  modules <- drop_landing(modules)
```

Comment:
Let's circle back to the naming topic. I would like the _landing page module_ or _popup module_ be an application of more general functionality _X_. `uiless`, `tabless` were discussed. Any other idea?

Reply:
Here we isolate the landing popup module and only one can be used, so I think the naming of the variable is apt. 
I don't mind generalizing the extract/drop functions at this time, though we are getting a little ahead of ourselves.